recipes/newrelic/apm/php/debian.yml

```diff
@@ -501,6 +511,10 @@ install:
               fi
 
               sed -i "s/newrelic.appname = \"[^\"]*\"/newrelic.appname = \"${APPLICATION_NAME}\"/" $ini_full_name
+          
+              LICENSE_KEY="{{.NEW_RELIC_LICENSE_KEY}}"
+              sed -i -E "s/newrelic.license = \"(REPLACE_WITH_REAL_KEY|)\"/newrelic.license = \"${LICENSE_KEY}\"/" $ini_full_name
+          
```

Comment:
In continuation to the explanation in the above comment, a different limitation noticed because of the aforementioned change in the agent was that the `newrelic.ini` file populated in the `mods-available` directory had no license key populated.

Upon discussing this with the PHP Agent, it was obtained that the cause of this is as follows. Earlier, since a new `newrelic.ini` would be written to one of `apache2` or `fpm` or `cli` by the recipe (as provided by the tarball installer, even in the event of a `newrelic.ini` existing in `mods-available`), the `newrelic.ini` file would be written with the license key, and our code in the recipe would move the `newrelic.ini` from one of these folders into `target_ini_dir`, i.e. `../../mods-available`, thereby making sure that the `newrelic.ini` in `mods-available` is overwritten with the `newrelic.ini` in one of these directories, with the license key, facilitating instrumentation.

However, now that the `newrelic.ini` file is no longer generated in any of these directories if there already exists a `newrelic.ini` in `mods-available`, the `newrelic.ini` in `mods-available` is not replaced by any other version of `newrelic.ini` elsewhere. 

Added to this is the fact that the `newrelic.ini` in `mods-available` tends to be generated without a license key.

A combination of all of these would mean we would need to `sed` our way into affixing the license key into the `newrelic.ini` that is now bound to stay in `mods-available`, but did not have the license key so far, and hence needs it; this is why this change is needed.